en/reference/document-v1-api-reference.html

```diff
@@ -30,290 +30,270 @@ <h2 id="configuration">Configuration</h2>
-          </p>
-          <ul>
-            <li>
-              Visit all documents in a group:
+      <p>
+        Documents can be grouped to limit accesses to a subset.
+        A group is defined by a numeric ID or string — see <a href="../documents.html#id-scheme">id scheme</a>.
+      </p>
 <pre>
 /document/v1/&lt;namespace&gt;/&lt;document-type&gt;/group/&lt;group&gt;
 /document/v1/&lt;namespace&gt;/&lt;document-type&gt;/number/&lt;number&gt;
 </pre>
-            </li>
-          </ul>
-        </td>
-      </tr>
-    </table>
-  </td>
-</tr><tr>
-  <th>POST</th>
-  <td>
-    <p id="post">
-      <em>Put</em> a given document, by ID,
-      or <em>Copy</em> a set of documents by selection from one content cluster to another.
-    </p>
-    <table class="table">
-      <tr>
-        <th>Put</th>
-        <td>
-          Write the document contained in the request body in JSON format.
+      Mandatory parameters:
+      <ul>
+        <li><a href="#cluster">cluster</a> -
+          Visits can only retrieve data from <em>one</em> content cluster,
+          so <code>cluster</code> <strong>must</strong> be specified
+          for requests at the root <code>/document/v1/</code> level, or when there is ambiguity.
+          This is required even if the application has only one content cluster.</li>
+      </ul>
+      Optional parameters:
+      <ul>
+        <li><a href="#bucketspace">bucketSpace</a> -
+          Parent documents are <a href="services-content.html#document">global</a>
+          and in the <code>global</code> <a href="#bucketspace">bucket space</a>.
+          By default, visit will visit non-global documents
+          in the <code>default</code> bucket space, unless document type is indicated,
+          and is a global document type.
+        </li>
+        <li><a href="#concurrency">concurrency</a></li>
+        <li><a href="#continuation">continuation</a></li>
+        <li><a href="#fieldset">fieldSet</a></li>
+        <li><a href="#selection">selection</a></li>
+        <li><a href="#timeout">timeout</a></li>
+        <li><a href="#tracelevel">tracelevel</a></li>
+        <li><a href="#wanteddocumentcount">wantedDocumentCount</a></li>
+      </ul>
+      <p>Examples:</p>
+      <ul>
+        <li>
+          Note that visit via selection is evaluated over all the stored documents in the cluster.
+          To split visiting of the document corpus across more than one HTTP
+          request—thus allowing the concurrent use of more HTTP containers—use the
+          <a href="#slices">slices</a> and <a href="#sliceId">sliceId</a> parameters.
+        </li>
+        <li>
+          It's also recommended enabling streamed HTTP responses,
+          with the <a href="#stream">stream</a> parameter,
+          as this reduces memory consumption and reduces HTTP overhead.
+          Finally, use the <a href="#concurrency">concurrency</a> parameter
```

Comment:
Only true when `stream` is _not_ used. When `stream=true`, concurrency _limits_ the maximum concurrency, which is otherwise unbounded, but controlled by a dynamic throttle policy.